raster_loader/io/common.py

```diff
@@ -347,13 +348,24 @@ def raster_band_stats(raster_dataset: rasterio.io.DatasetReader, band: int) -> d
             else:
                 (raw_data, mask) = band_with_nodata_mask(raster_dataset, band)
             stats = np.ma.masked_array(data=raw_data, mask=mask)
+    qdata = stats.compressed()
+    ranges = [[j / i for j in range(1, i)] for i in range(3, 20)]
+    quantiles = [
+        [int(np.quantile(qdata, q, method="lower")) for q in r] for r in ranges
+    ]
+    quantiles = dict(zip(range(3, 20), quantiles))
```

Comment:
This look good to me. Should we have the range 3-20 in a const variables? In any case LGTM

Reply:
will add :)